test/framework/options.py

```diff
@@ -3048,6 +3051,26 @@ def test_check_style(self):
         for pattern in patterns:
             self.assertTrue(re.search(pattern, stdout, re.M), "Pattern '%s' found in: %s" % (pattern, stdout))
 
+    def test_allow_use_as_root(self):
+        """Test --allow-use-as-root"""
+
+        # pretend we're running as root by monkey patching os.getuid used in main
+        easybuild.main.os.getuid = lambda: 0
```

Comment:
some linting tools don't like it when you assign lambda's, but this one is so trivial it doesn't require a def imho.